wiki/Beatmap/Marathon/de.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Marathon
+
+Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mehr als fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```

Comment:
```suggestion
Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mindestens fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```

Reply:
To not have `mindestens` twice in the article, you could also write:
```suggestion
Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) zu beschreiben, die länger als 5 Minuten sind. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
```